templates/deployments.yaml

```
@@ -52,7 +52,7 @@ spec:
         {{- end }}
         envFrom:
         - secretRef:
-            name: {{ .Release.Name }}-st2-auth
+            name: {{- include "stackstorm-ha.secretSt2Auth" . }}
```

```suggestion
            name: {{ include "stackstorm-ha.secretSt2Auth" . }}
```

I don't think you need the `-` on these lines, do you?

---

You are correct. After adding the additional testing, I found that that actually caused issues.